src/components/PersonTabs/PeopleTabs.tsx

```diff
@@ -0,0 +1,121 @@
+import { Loader } from '../Loader';
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import classNames from 'classnames';
+import { PersonLink } from '../PersonLink';
+import { useParams } from 'react-router-dom';
+
+export const PeopleTabs = () => {
+  const [peoples, setPeoples] = useState<Person[]>([]);
```

Comment:
Consider renaming 'peoples' to 'people' for better readability and to match the common plural form of 'person'.